src/Support/SelectFields.php

```diff
@@ -390,6 +366,116 @@ private static function isMongodbInstance(GraphqlType $parentType): bool
         return isset($parentType->config['model']) ? app($parentType->config['model']) instanceof $mongoType : false;
     }
 
+    /**
+     * @param array $queryArgs
+     * @param array $field
+     * @param GraphqlType $parentType
+     * @param array $select
+     * @param array $with
+     * @param mixed $ctx
+     * @param FieldDefinition $fieldObject
+     * @param string $key
+     * @param Closure|null $customQuery
+     */
+    protected static function handleInterfaceFields(
+        array $queryArgs,
+        $field,
+        GraphqlType $parentType,
+        array &$select,
+        array &$with,
+        $ctx,
+        $fieldObject,
+        $key,
+        $customQuery
+    ) {
+        $relationsKey = Arr::get($fieldObject->config, 'alias', $key);
+
+        $with[$relationsKey] = function ($query) use ($queryArgs, $field, $parentType, &$select, $ctx, $customQuery, $key, $fieldObject) {
+            $parentTable = self::isMongodbInstance($parentType) ? null : self::getTableNameFromParentType($parentType);
+
+            self::addForeignKey($select, $query, $parentTable, $field);
+
+            // New parent type, which is the relation
+            try {
+                if (method_exists($parentType, 'getField')) {
+                    $newParentType = $parentType->getField($key)->config['type'];
+                } else {
+                    return $query;
+                }
+            } catch (InvariantViolation $e) {
+                return $query;
+            }
+
+            self::addAlwaysFields($fieldObject, $field, $parentTable, true);
+
+            if ($newParentType instanceof WrappingType) {
+                $newParentType = $newParentType->getWrappedType(true);
+            }
+
+            if (isset($newParentType->config['types'])) {
+                foreach ($newParentType->config['types']() as $type) {
+                    if (app($type->config['model'])->getTable() === $query->getRelated()->getTable()) {
```

Comment:
For each types of the interface, we are fetching the table of the model. If it is the same table as the one we are currently querying from, then we should build our custom query, and execute it. (`return $callable($query);`)